src/exposurelog/routers/delete_message.py

```diff
@@ -11,11 +13,11 @@
 router = fastapi.APIRouter()
 
 
-@router.delete("/messages/{id}", status_code=204)
+@router.delete("/messages/{id}", status_code=http.HTTPStatus.NOT_FOUND)
```

Comment:
I'm curious why you chose the default status code to be NOT_FOUND rather than NO_CONTENT